pages/test_analytics.md

```diff
@@ -21,7 +21,9 @@ Test Analytics helps you track and analyze the steps in that pipeline that invol
 <div class="ButtonGroup">
   <%= button ":rspec: RSpec", "/docs/test-analytics/ruby-collectors#rspec-collector" %>
   <%= button ":ruby: minitest", "/docs/test-analytics/ruby-collectors#minitest-collector" %>
-  <%= button ":jest: Jest", "/docs/test-analytics/javascript-collectors" %>
+  <%= button ":jest: Jest", "/docs/test-analytics/javascript-collectors#jest" %>
```

Comment:
I tested this link to see if it scrolled to the correct position on the page but it didn't work. It looks like the ID of these heading elements are `#configure-the-test-framework-jest`, `#configure-the-test-framework-mocha-collector`, `#configure-the-test-framework-jasmine-collector`

Reply:
😬 This is weird, it's working on my local

Reply:
Nvm, I reverted back to the old heading styles and the above works

Reply:
So weird! I've found anchor scrolling a bit buggy in the past as well 🤷